Source/SwiftLintFramework/Rules/ValidIBInspectableRule.swift

```diff
@@ -118,7 +118,7 @@ public struct ValidIBInspectableRule: ASTRule, ConfigurationProviderRule {
         ]
 
         let intTypes: [String] = ["", "8", "16", "32", "64"].flatMap { size in
-            ["U", ""].flatMap { (sign: String) -> String in
+            ["U", ""].map { (sign: String) -> String in
```

Comment:
`map`, instead of `flatMap/compactMap` could be used on line 120 too.

Reply:
No, it can't.
That is [`func flatMap<SegmentOfResult>(_ transform: (Self.Element) throws -> SegmentOfResult) rethrows -> [SegmentOfResult.Element] where SegmentOfResult : Sequence`](https://developer.apple.com/documentation/swift/sequence/2905332-flatmap).